packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -84,19 +84,7 @@ export function PanelChrome({
 }: PanelChromeProps) {
   const theme = useTheme2();
   const styles = useStyles2(getStyles);
-
-  // To Do rely on hoverHeader prop for header, not separate props
-  // once hoverHeader is implemented
-  //
-  // Backwards compatibility for having a designated space for the header
-
-  const hasHeader =
-    hoverHeader === false &&
-    (title.length > 0 ||
-      titleItems !== undefined ||
-      description !== '' ||
-      loadingState === LoadingState.Streaming ||
-      (leftItems?.length ?? 0) > 0);
+  const hasHeader = !hoverHeader;
```

Comment:
🤔 Wouldn't this change break backward compatibility with plugins that rely on not showing a header if there is no title?

Let's say I am using the `PanelChrome` component with the old API where only `title` and `leftItems` were available.

In the old behavior, if the user did not pass both of these props, there was no header. However, with this change, I think we will get a header because hoverHeader=false on line 78."

Reply:
True, i considered checking if hoverHeader was undefined and then revert to old behavior.


But not sure it's worth it, doubt any plugin rely on this old behavior, and it's such small breaking behavior

Reply:
I did a quick search of `PanelChrome` usages and found a few of our internal app plugins are using the old API of `PanelChrome` (slo, ml-app, incidents)  

Taking a look at how they used it, I found [this](https://github.com/grafana/slo/blob/9864a136dd126ec6c5f5508f214387a39d60cecd/src/components/Vizualizer/ResponseVisualizer.tsx#L41) example, in this case having an extra header, will break their original intent. 

I think we should do our best to keep our components backward compatible between minor versions, saying that, this change can be added to Grafana10 breaking changes :) .

Reply:
@axelavargas no it wont, that example will look identical to today, it has a title="SLO"

Reply:
not sure what you mean "extra header", the SLO example should look identical to this PR, no change extra header?

Reply:
> @axelavargas no it wont, that example will look identical to today, it has a title="SLO"

I think I am failing to understand the logic here 😅 ,  this is what I am understanding:

1. we have this piece of code

```
<PanelChrome width={width} height={height}>
    {(innerWidth, innerHeight) => (
      <PanelRenderer
        title="SLO"
        pluginId="timeseries"
        width={innerWidth}
        height={innerHeight}
        data={data}
        options={panelOptions}
        fieldConfig={fieldConfig}
      />
    )}
 </PanelChrome>
```

2. `panelRenderer` is the child of our `PanelChrome` and it is responsible to render `timeseries` panel
3.  Then we have the `title="SLO"` that belongs to timeseries component, so when you say "that example will look identical to today", do you mean that timeseries component is also controlling the header of our `PanelChrome`? 


> not sure what you mean "extra header", the SLO example should look identical to this PR, no extra header?

Yes, sorry I did not use the right words 🙈 , I mean having a header when we do not have `title` and `leftitems`. 

![image](https://user-images.githubusercontent.com/239999/223088119-8b8a6354-330b-4d50-b285-3eb3e10bf3fd.png)

Reply:
@axelavargas I think the SLO app is not launched yet and they will switch to scenes soon anyway before launch, is there any other usage?

Reply:
Yes, I found a few usages on [ml-app](https://github.com/grafana/grafana-ml-app/blob/e4d62007a01e4401846a5a82313f1a32d4e072e7/src/components/JobGraph/JobGraphContent.tsx#L70) ,  [incidents](https://github.com/grafana/incident/blob/2c39632f845fe784d3254266e6131db625e1aefa/plugin/src/components/Graph.tsx#L138), but in these cases, I see they are using the old `leftItems`, that will add that header space. Other usages were on hackathon projects, but I don't think we should worry about them.

I think we can keep your suggestion, and put the effort into creating the MDX doc for existing plugins to migrate to the new Panel API :)